src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java

```diff
@@ -332,6 +332,56 @@ public boolean siVisualizzaUnaDelegaPG() {
         }
     }
 
+    public void checkAlertWrongDelegationCode() {
+        try {
+            By alertWrongCode = By.id("alert-api-status}");
+            By alertCloseButtonBy = By.xpath("//*[@aria-label='Close']");
```

Comment:
Maybe this can be used instead? 
https://github.com/pagopa/pn-frontend-e2e/blob/ffdbfbd7bf209f0ae62e5056c13c9a861891495e/src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java#L208-L219

Reply:
I created the method cause i need to close the alert pop-up, otherwise there can be problems in the next phase of the test.
Furthermore i don't know why the id has a "}" at the end, is it a little typo?

Reply:
If it is used somewhere else maybe it's better to declare it global at the top of the file.
The `}` has already been addressed on a previous PR https://github.com/pagopa/pn-frontend-e2e/pull/117#discussion_r1514610615 and will be fixed in the future updates: